magicctapipe/io/tests/test_io_monly.py

```diff
@@ -333,12 +333,12 @@ def test_get_stereo_events_data(stereo_monly):
         event_data = pd.read_hdf(str(file), key="events/parameters")
         event_data.set_index(["obs_id", "event_id", "tel_id"], inplace=True)
         event_data.sort_index(inplace=True)
-        data = get_stereo_events(event_data)
-        assert np.all(data["multiplicity"] == 2)
-        assert np.all(data["combo_type"] == 0)
+        data = get_stereo_events(event_data, config_gen)
+        assert np.all(data["multiplicity"] == 2)    
```

Comment:
some unneeded extra spaces (you could run a linter over modified files to do it automatically, but this might result in multiple irrelevant changes if it was not run before on the file)

Reply:
We have to run black, but Alessio (?) suggested me to run it only after the review to simplify the reviewing process

Reply:
as you prefer. but those things also end up in the review as fake changes
so do you want me to close all those comments where I pointed out unnecessary spaces that show up as change of code?

Reply:
I think we can do the stylistic changes (unneeded spaces, blank lines etc) in the PR I already opened #153 . That is a PR just for that, plus making the CI fail if the code does not pass the linting phase. So people will be forced to run a linter to make CI pass (I will provide instructions, actually to be added in that PR in the README, will do that).